src/webgpu/api/validation/vertex_state.spec.ts

```diff
@@ -55,22 +38,22 @@ Test that an attribute must be aligned to the component size:
      - Check error IFF offset not aligned to componentsize(format);
 `;
 
+import { params, pbool, poptions } from '../../../common/framework/params_builder.js';
 import { makeTestGroup } from '../../../common/framework/test_group.js';
 
 import { ValidationTest } from './validation_test.js';
 
 const MAX_VERTEX_ATTRIBUTES: number = 16;
 const MAX_VERTEX_BUFFER_END: number = 2048;
 const MAX_VERTEX_BUFFER_ARRAY_STRIDE: number = 2048;
-const MAX_VERTEX_BUFFERS: number = 16;
+const MAX_VERTEX_BUFFERS: number = 8;
```

Comment:
Consider moving all of these constants into capability_info (and renaming to kMaxVertexBuffers etc). BTW the type annotation is unnecessary.

Reply:
I'll do that as a follow-up.